packages/api/amplify_api/test/amplify_api_config_test.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:convert';
+
+import 'package:amplify_api/src/amplify_api_config.dart';
+import 'package:amplify_core/amplify_core.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'test_data/fake_amplify_configuration.dart';
+
+void main() {
+  late EndpointConfig endpointConfig;
+
+  group('GraphQL Config', () {
+    const endpointType = EndpointType.graphQL;
+    const endpoint =
+        'https=//abc123.appsync-api.us-east-1.amazonaws.com/graphql';
+    const region = 'us-east-1';
+    const authorizationType = APIAuthorizationType.apiKey;
```

Comment:
nit: if mocking api key type might as well add a fake api key here in case there was some code path that assumed those things connected (though that's not the case here).